src/components/AvailabilityGrid.tsx

```diff
@@ -74,29 +135,47 @@ const Grid = ({
     the mouse button is pressed 
   */
   const handleMouseEnter = (rowIndex: number, colIndex: number) => {
-    if (!isAvailable) return
+    if (!isAvailable) {
```

Comment:
have you run prettier?

Reply:
because the indentation looks off.

Reply:
i did run prettier and it did pass the check. Will double check on this